api/test/integration/test_security_DELETE_endpoints.tavern.yaml

```
@@ -137,7 +137,15 @@ stages:
     response:
       status_code: 200
       json:
-        <<: *error
+        error: 1
+        data:
+          affected_items: []
+          failed_items:
+            - error:
+                code: 4008
+              id: !anything
```

```suggestion
              id: 999
```

In addition, please specify in the test name that this policy is a default policy. That is why we get an error 4008 and not a 4007.